packages/flutter_tools/README.md

```diff
@@ -2,6 +2,32 @@
 
 Tools for building Flutter applications.
 
+## Setup
```

Comment:
I am surprised that you had to do all these things. Executing any `flutter` command (e.g. `flutter doctor`) should download the dart sdk and execute pub get in flutter_tools automatically for you. There shouldn't be a need for doing this manually.

The third step (pub get in asset_test) is performed by `flutter update-packages`.

In short: It might make more sense to just instruct people to execute `flutter update-packages` which would do all these steps listed here in one convenient step.